csg.js

```diff
@@ -1983,11 +1983,11 @@ for solid CAD anyway.
                                 this._z = 0;
                             }
                         }
-                    } else if (('_x' in x) && ('_y' in x)) {
-                        this._x = parseFloat(x._x);
-                        this._y = parseFloat(x._y);
-                        if ('_z' in x) {
-                            this._z = parseFloat(x._z);
+                    } else if (('x' in x) && ('y' in x)) {
```

Comment:
These fields names changes are necessary to allow CAG to be stored as anonymous objects, and then restored from those via CAG.fromObject(). Basically, this fixes CAG.fromObject().